auth-api/src/auth_api/version.py

```diff
@@ -22,4 +22,4 @@
 Development release segment: .devN
 """
 
-__version__ = '0.1.0a0.dev'  # pylint: disable=invalid-name
+__version__ = '0.2.0a0'  # pylint: disable=invalid-name
```

Comment:
I am not sure if we need to do this.. any particular reason for doing this?

Reply:
Yeah...I am aware of that..
We release a bit different from entities ;mostly big release we do with bunch of tickets together..We don't generally rollout one change into prod quickly..mostly big releases...So we don't have this practice so far..but i am okay to bring in this practice ...May be we can discuss with @pwei1018 and agree on it..

Reply:
fixed version and a lint error and repushing now